.ci/install_runtime.sh

```diff
@@ -116,12 +116,12 @@ fi
 
 if [ "${TEST_CGROUPSV2}" == "false" ]; then
 	case "${KATA_HYPERVISOR}" in
-		"cloud-hypervisor" | "qemu")
-			echo "Add kata-runtime as a new/default Docker runtime."
```

Comment:
For clarification, `manage_ctr_mgr.sh" docker configure -r kata-runtime -f` will not override default runtime as this new runtime(here, `kata-runtime`), the default one will still be `runc`, except it is set for `docker swarm`
see https://github.com/kata-containers/tests/blob/master/cmd/container-manager/manage_ctr_mgr.sh#L299